clients/fides-js/src/lib/tcf/constants.ts

```diff
@@ -48,6 +51,12 @@ export const TCF_KEY_MAP: {
     tcfModelKey: "vendorLegitimateInterests",
     enabledIdsKey: "vendorsLegint",
   },
+];
+
+export const TCF_COOKIE_KEY_MAP: {
+  cookieKey: TcfModelType;
+  experienceKey: keyof TcfExperienceRecords;
```

Comment:
it doesn't seem like anything uses it, probably since we combine `vendors` with `systems` for enabledIds. this variable should maybe be a different name, but I haven't thought of a great one yet. maybe `FIDES_SYSTEM_COOKIE_KEY_MAP`?